.github/workflows/nodejs.yml

```diff
@@ -13,6 +13,7 @@ jobs:
     - run: npm ci --no-audit
     - run: npm run lint --if-present
     - run: npm test
+    - run: npm run make-badges --if-present
```

Comment:
It is an interesting idea to add the badge script to GH Actions, but I think it won't add the new badge to repo without a git add, commit, and push.  Adding that would be more complicated.  Maybe for now, just add the badge making to the `test` script (in package.json)?

Reply:
That is true, I removed the action. Added a single `test` script to run tests and then, generate the badge .